VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -48,11 +46,12 @@ export class AppComponent implements OnInit, OnDestroy {
         private connectionStatusService: ConnectionStatusService,
         pageTracker: PageTrackerService,
```

Comment:
any reason why page tracker and language service are not private?

Reply:
only used in ctor